src/test/java/org/apache/commons/collections4/bloomfilter/DefaultBloomFilterTest.java

```diff
@@ -67,6 +68,110 @@ public void testHasherBasedMergeWithDifferingSparseness() {
                 .forEachBitMapPair(bf1, (x, y) -> x == y));
     }
 
+    @Test
+    public void testEstimateNWithBrokenCardinality() {
+        // build a filter
+        BloomFilter filter1 = TestingHashers.populateEntireFilter(new BrokenCardinality(getTestShape()));
+        assertThrows(IllegalArgumentException.class, ()->filter1.estimateN());
+    }
+
+    @Test
+    public void testLargeBitmapAsArray() {
```

Comment:
I do not know why we need this test. It is just checking that the `asArray` function works for a maximum length filter? If this is the purpose then can we just use a filter with `bitIndex = Integer.MAX_VALUE` set to 1, e.g. by using a hasher with 1 index to populate the filter.

Is this code simply to test a fully populated max length filter, e.g. it was debugging code?